cleanlab/models/keras.py

```diff
@@ -190,7 +196,9 @@ def fit(self, X, y=None, **kwargs):
         self.net.fit(X, **kwargs)
 
     def predict_proba(self, X, *, apply_softmax=True, **kwargs):
-        """Set extra argument `apply_softmax` to True to indicate your network only outputs logits not probabilities."""
+        """Predict class probabilities ``P(true label=k)`` using the wrapped Keras classifier.
```

Comment:
why are we saying P(true label ) here?

More generally why isn't first sentence from all these docstrings just identical to say the first sentence of the analogous docstring from sklearn?

Reply:
The `P(true label = k)` part was taken from the [CleanLearning docstrings](https://github.com/cleanlab/cleanlab/blob/master/cleanlab/classification.py#L609), I can change it to make it more similar to sklearn. 

Also if the `P(true label = k)` part is confusing should we also edit it for the CleanLearning documentation?

Reply:
Ah there is a difference in that CleanLearning is explicitly intended to estimate `P(true label = k)` rather than `P(given label = k)` but most models will just estimate the latter.

So here we should just imitate a combination of sklearn & keras docstrings, since this module is not about training on cleaned data like CleanLearning